dashboard-server/queries/responses/response.queries.js

```diff
@@ -33,8 +36,9 @@ async function _all({ email, survey, timestamp, userid, ref, pageSize }, pool) {
   ORDER BY (timestamp, responses.userid, question_ref)
   LIMIT $6`;
 
+  pool = new Pool(DATABASE_CONFIG);
```

Comment:
Wait, no, this shouldn't be necessary! Look at how all the other functions in the controllers access pool. Follow the same pattern as the other developers - find the pattern!

Reply:
@sssoja just pinging you on this.